CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### 🧰 Bug fixes 🧰
 
+- `kafkametricsreceiver`: Do not crash collector if Kafka is unavailable during startup (#8817)
```

Comment:
Please capture the change in behaviour here. The client connection isn't configured at startup, but rather on first scrape.